### PR TITLE
Remove read_file_to_str helper

### PR DESCRIPTION
**Pull request by mxpv**
Use `fs::read_to_string` from stdlib/tokio instead.

### CODE REVIEW

File: crates/shim/src/error.rs [hunk 1]
```diff
@@ -84,6 +84,9 @@ pub enum Error {
     #[error("Failed to send exit event: {0}")]
     Send(#[from] std::sync::mpsc::SendError<ExitEvent>),
 
+    #[error("Failed to read file: {0}")]
+    Io(#[from] std::io::Error),
```
Comment (jsturtevant) [inline crates/shim/src/error.rs:88]: Will this type of error always be `failed to read a file`?

**Review by jsturtevant — APPROVED** [review]
LGTM

**Review by Burning1020 — CHANGES_REQUESTED** [review]

File: crates/shim/src/error.rs (hunk 1 above)
Comment (mxpv) [reply crates/shim/src/error.rs:88]: I don't really like the requirement to map errors and use macros everywhere, like
`map_err(io_error!`. I feel like [`anyhow.Context`](https://docs.rs/anyhow/latest/anyhow/trait.Context.html#example) would be a better fit here.

Comment (Burning1020) [reply crates/shim/src/error.rs:88]: However, anyhow error is not easy to downcast if you need check the exact type of error

Comment (mxpv) [reply crates/shim/src/error.rs:88]: Where exactly we need to downcast?

Comment (Burning1020) [reply crates/shim/src/error.rs:88]: > Where exactly we need to downcast?

When you want to ignore some specific error.

**Pull request closed without merging**